views/js/qtiRunner/core/Renderer.js

```diff
@@ -753,6 +753,10 @@ define([
                 return this._areaBroker;
             }
         };
+
+        this.getItemCreator = function getItemCreator() {
+            return this.getOption('itemCreator');
+        };
```

Comment:
I am not we really need this item creator specific method in the core/Renderer as it is used only by a specific renderer. If you really want to add it, you should propably add this where the creatorRenderer is build or just alloy the QTI creator to call directly this option `this.getOption('itemCreator')`